demo/src/modules/home/containers/TableContainer.js

```diff
@@ -0,0 +1,218 @@
+import React, { useState, useEffect } from 'react';
+import { DataTable } from 'asab-webui';
+import { Container } from 'reactstrap';
+
+export default function (props) {
+  const headers = [ { name: 'name', type: 'string' }, { name: 'provider', type: 'string' }, { name: 'type', type: 'string' }, { name: ' ', type: 'Json' }, ];
```

Comment:
@titkov49 Please, use tabs instead of spaces in the containers:
demo/src/modules/home/containers/TableContainer.js
src/containers/DataTable/Pagination.js
src/containers/DataTable/index.js 

And please add a newline at the end of the files